Cargo.toml

```diff
@@ -66,9 +66,24 @@ default = ["unproven"]
 unproven = ["embedded-hal/unproven"]
 device-selected = []
 direct-call-deprecated = []
+ld = []
 rt = ["stm32f3/rt"]
 can = ["embedded-hal-can"]
 
+svd-f301 = ["stm32f3/stm32f301"]
+svd-f302 = ["stm32f3/stm32f302"]
+svd-f303 = ["stm32f3/stm32f303"]
+svd-f373 = ["stm32f3/stm32f373"]
+svd-f3x4 = ["stm32f3/stm32f3x4"]
+
+mem-4 = []
```

Comment:
Out of curiosity, where to have the namings from? Can you point me towards the according documentation?

Reply:
Well, 4/6/8/b/c/d/e is just came from part number

Reply:
I see 😄